plugins/modules/backupunit.py

```diff
@@ -306,6 +321,19 @@ def get_sdk_config(module, sdk):
 
 
 def check_required_arguments(module, state, object_name):
+    # manually checking if token or username & password provided
+    if (
+        not module.params.get("token") and
+        not module.params.get("username") or not module.params.get("password")
```

Comment:
think there need to be another set of parentheses here as I think and is evaluated before or

I think writing it like this is easier to understand
if (
not module.params.get("token")
and not ( module.params.get("username") and module.params.get("password") )
):